ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v1/container/clientlibs/editor/js/editDialog.js

```diff
@@ -419,6 +420,31 @@
         }
     }
 
+    function showHideAutoSave(dialog) {
+        var enableAutoSave = dialog.find(ENABLE_AUTO_SAVE);
+        if (enableAutoSave[0]) {
+            var autoSaveStrategyContainer = dialog.find('.autoSaveStrategyContainer');
```

Comment:
all dialog should be accessed via BEM style class, this does not align to BEM

Reply:
Done